Sources/IO/Core/HttpSceneLoader/index.js

```diff
@@ -213,10 +218,33 @@ function loadHttpDataSetReader(item, model, publicAPI) {
 
 // ----------------------------------------------------------------------------
 
+function loadTimeStepBasedAnimationHandler(
+  data,
+  model,
+  publicAPI,
+  setCameraParameters,
+  setBackground
+) {
+  model.animationHandler = vtkTimeStepBasedAnimationHandler.newInstance({
+    scene: model.scene,
+    originalMetadata: model.metadata,
+    applySettings,
+    setCameraParameters,
+    setBackground,
```

Comment:
Should `setCameraParameters` and `setBackground` replaced by `renderer` and `renderWindow`?

Reply:
We don't even need the renderWindow in this case